rpcclient/infrastructure.go

```diff
@@ -1097,6 +1097,22 @@ func (c *Client) sendRequest(jReq *jsonRequest) {
 // future.  It handles both websocket and HTTP POST mode depending on the
 // configuration of the client.
 func (c *Client) SendCmd(cmd interface{}) chan *Response {
+	return c.sendCmdWithPriority(cmd, false)
+}
+
+// SendCmdLazy flags the the passed command as low priority and sends it to the
+// associated server and returns a response channel on which the reply will be
+// delivered at some point in the future. It handles both websocket and HTTP
+// POST mode depending on the configuration of the client.
+func (c *Client) SendCmdLazy(cmd interface{}) chan *Response {
```

Comment:
Low prio: To me, the word "lazy" in programming indicates "don't do it immediately, only when we actually need the result". Which kind of fits, but also not fully. Wasn't able to come up with a better suffix though, so I asked ChatGPT. Not sure if any of those fit better:
```
SendCmdLight - Using "Light" suggests a less intensive or lower priority operation.
SendCmdLazy - "Lazy" often implies that the action isn't urgently prioritized.
SendCmdIdle - "Idle" might indicate that the task is of low urgency.
SendCmdSlow - Directly indicates that it's a less priority-sensitive function.
SendCmdBack - As in "backburner", where it's not the immediate focus.
SendCmdMinor - Signifies lesser importance without being too long.
SendCmdRelax - Suggests that the function can "relax" and doesn't need to hurry.
SendCmdSoft - A softer version, possibly less critical.
SendCmdCasual - Implying the task is non-urgent.
SendCmdChill - A colloquial way to denote low urgency.
```

Reply:
Maybe `SendCmdWithLowPriority`? kinda long...

Reply:
Yeah, it's a bit long. My favorite from the list is actually `SendCmdSlow`.

Reply:
switched to `slow`